### ISSUE TITLE
Apply for submitting A Chineses version of markdown

### ISSUE THREAD

**Issue by culome** [PuerkitoBio/gocrawl#80]
Hello, I'm interested in gocrawl and I want to submit Chinese version of translation of readme. What can I do ? Can I give a PR on that?

**Comment by mna** [comment]
Yeah sure, a PR with the translation in a separate markdown file, and a
link to the translation from the readme would be great!
